include/aws/common/math.fallback.inl

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <aws/common/exports.h>
+#include <stdint.h>
+#include <stdlib.h>
```

Comment:
Because all of these inl files are included by math.h, which already includes common.h, and stdint.h and stdlib.h, you could probably just remove all of these includes.

This will also help in the case we're on a hella old GCC install without stdint.h, in which case we define our own int types in common.h

Reply:
This is mostly to make my editor shut up and stop redlining all the uint64_ts.

Reply:
I'll replace with common.h though